docs/Manage-Space.rst

```diff
@@ -273,6 +273,8 @@ Type-ahead suggestions allow to facilitate selection users and groups to invite.
 
 .. note:: Only spaces that the creator is member of can be selected.
 
+.. note:: You can have predefined space members or users in the Users field.
```

Comment:
```suggestion
.. note:: You can have predefined space members or users in the Users field.
```

Reply:
Users field can be pre-filled by default users or spaces defined in space template.